### PR TITLE
[Compose] Update to beta07

### PR DESCRIPTION
**Pull request by manuelvicnt**

### CODE REVIEW

**Review by yrezgui — APPROVED** [review]

File: app/src/main/java/com/google/samples/apps/sunflower/compose/plantdetail/PlantDetailView.kt [hunk 1]
```diff
@@ -245,37 +249,44 @@ private fun PlantImage(
...
+        Image(
+            painter = painter,
+            contentDescription = null,
+            contentScale = ContentScale.Crop,
+            modifier = Modifier.fillMaxSize(),
+        )
+        if (painter.loadState is ImageLoadState.Loading) {
             Box(
                 modifier = Modifier
                     .fillMaxSize()
```
Comment (chrisbanes) [inline app/src/main/java/com/google/samples/apps/sunflower/compose/plantdetail/PlantDetailView.kt:267]: Nit: should probably be matchParentSize() instead

File: app/src/main/java/com/google/samples/apps/sunflower/compose/plantdetail/PlantDetailView.kt [hunk 2]
```diff
@@ -245,37 +249,44 @@ private fun PlantImage(
...
+        modifier = modifier
+            .fillMaxWidth()
+            .height(Dimens.PlantDetailAppBarHeight)
+    ) {
+        val painter = rememberCoilPainter(request = imageUrl, fadeIn = true)
+        Image(
+            painter = painter,
+            contentDescription = null,
+            contentScale = ContentScale.Crop,
+            modifier = Modifier.fillMaxSize(),
```
Comment (chrisbanes) [inline app/src/main/java/com/google/samples/apps/sunflower/compose/plantdetail/PlantDetailView.kt:262]: Nit: should probably be matchParentSize() instead

File: app/src/main/java/com/google/samples/apps/sunflower/compose/plantdetail/PlantDetailView.kt [hunk 3]
```diff
@@ -245,37 +249,44 @@ private fun PlantImage(
...
 private fun PlantFab(
     onFabClick: () -> Unit,
     modifier: Modifier = Modifier
 ) {
+    val addPlantContentDescription = stringResource(R.string.add_plant)
     FloatingActionButton(
         onClick = onFabClick,
         shape = MaterialTheme.shapes.small,
-        modifier = modifier
+        modifier = modifier.semantics {
```
Comment (chrisbanes) [inline app/src/main/java/com/google/samples/apps/sunflower/compose/plantdetail/PlantDetailView.kt:283]: Why the change of using `semantics` on the parent?

Comment (manuelvicnt) [reply app/src/main/java/com/google/samples/apps/sunflower/compose/plantdetail/PlantDetailView.kt:283]: This is a bug that came in `beta04`. Adding the bug as a comment, thanks!

File: app/src/main/java/com/google/samples/apps/sunflower/compose/plantdetail/PlantDetailView.kt (hunk 2 above)
Comment (manuelvicnt) [reply app/src/main/java/com/google/samples/apps/sunflower/compose/plantdetail/PlantDetailView.kt]: Done, thanks!

File: app/src/main/java/com/google/samples/apps/sunflower/compose/plantdetail/PlantDetailView.kt (hunk 1 above)
Comment (manuelvicnt) [reply app/src/main/java/com/google/samples/apps/sunflower/compose/plantdetail/PlantDetailView.kt]: Done, thanks!

**Review by chrisbanes — APPROVED** [review]
